### PR TITLE
Release 2.21.0

### PR DESCRIPTION
**Pull request by JustinTullgren**
AARs are still building. Since those aren't reviewed I'll start this PR for the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,7 +199,7 @@ one.optOut(true);
 *Note:* 
 - When opted out, tracking will stop and locally queued data will be removed.
 - At any point you can opt a user back in by passing `false` into the same method. 
-- For instructions on how completely remove a user's data from ONE - see our [one-api documentation](https://thunderheadone.github.io/one-api/#operation/delete).
+- For instructions on how completely remove a user's data from ONE - see our [api documentation](https://thunderheadone.github.io/one-api/#operation/delete).
```
Comment (tothstephen) [inline README.md]: Do we still need to reference ONE here?

Comment (JustinTullgren) [reply README.md]: yup need to change it.

File: README.md [hunk 2]
```diff
@@ -137,9 +151,9 @@ If you haven’t done so already, you will need to subclass your `Application` c
 
 ### Initialize the SDK
 #### Set up the Framework in User mode
-To start tracking, capturing, and receiving optimizations with the ONE SDK in User mode, you must first initialize it with your ONE API parameters. You can find your ONE API parameters on the ONE website.
+To start tracking, capturing, and receiving optimizations with the Thunderhead SDK in User mode, you must first initialize it with your Thunderhead API parameters. You can find your Thunderhead API parameters on the Thunderhead ONE website or in Salesforce Interaction Studio.
```
Comment (aherod20) [inline README.md]: is the website technically the ONE Engagement Hub?

Comment (JustinTullgren) [reply README.md]: Yes I guess. Got these values from @andreipop though.  Thoughts?

**Pull request merged**